docs/markdown/snippets/kconfig_enhancements.md

```diff
@@ -0,0 +1,3 @@
+## Enhancements to the kconfig module
+
+`kconfig.load()` may now accept a `configure_file()` input file.
```

Comment:
That should probably be "output file" rather than "input file"?

Reply:
eheh, it depends which side you look at it, I think both are correct, I don't mind changing it.

Reply:
`kconfig.load()` may now accept a `configure_file()` **as** input file.

Reply:
yep, thanks